templates/partial/_careers-available-roles.html

```diff
@@ -6,15 +6,14 @@
     <div class="col-6">
       <form action="" method="get" class="p-form p-form--inline">
         <div class="p-form__group">
-          <label for="full-name-inline" aria-label="Filter the resources by type" class="p-form__label">Filter by</label>
-          <select class="js-submit-on-change p-form__control" name="content" aria-label="Filter by content type">
-            <option value="">All</option>
-            <option value="">Home based</option>
-            <option value="">Office based</option>
-            <option value="">Management</option>
-            <option value="">Senior roles</option>
-            <option value="">Full-time</option>
-            <option value="">Part-time</option>
+          <label for="filter" aria-label="Filter by job type" class="p-form__label">Filter by</label>
+          <select class="js-submit-on-change p-form__control" id="filter" name="content" aria-label="Filter by job type">
+            <option {% if not category %}selected="selected"{% endif %} value="{{request.path}}#available-roles">All</option>
+           <option {% if category and category == 'home-based' %}selected="selected"{% endif %} value="{{request.path}}?category=home-based#available-roles">Home based</option>
+           <option {% if category and category == 'office-based' %}selected="selected"{% endif %} value="{{request.path}}?category=office-based#available-roles">Office based</option>
+           <option {% if category and category == 'management' %}selected="selected"{% endif %} value="{{request.path}}?category=management#available-roles">Management</option>
```

Comment:
Why do you need to put the full URL in the `value`? Wouldn't just the query string suffice? I believe `?something=else` should be a valid `href` that a browser would follow.